src/middleware/packages/webid/service.js

```diff
@@ -100,27 +97,33 @@ const WebIdService = {
         accept: accept
       });
     },
+
     async findByEmail(ctx) {
-      const { email } = ctx.params;
+      return await ctx.call('webid.findByFoafEmail', ctx.params);
```

Comment:
Un profil webID est toujours décrit selon l'ontologie FOAF non ? Je ne vois pas l'intérêt de renommer cette action.

Reply:
FOAF est l'usage, pas la norme
"FOAF is one fairly-commonly used WebID-centered vocabulary" : https://www.w3.org/wiki/WebID
"WebIDs can be used to build a Web of trust using vocabularies such as FOAF [FOAF]" : https://dvcs.w3.org/hg/WebID/raw-file/tip/spec/identity-respec.html

Reply:
par contre je suis d'accord pour fournir une méthode qui recherche selon le prédicat faof:email